### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by FinalAngel**
align with aldryn bootstrap boilerplate 3.4.0

### CODE REVIEW

File: private/sass/mixins/_functions.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// em calculation
+@function em($target, $context: $font-size) {
+    @if $target == 0 {
+        @return 0
+    }
+    @return $target / $context + em;
+}
+
+// line height calculation
+@function lh($line-height, $line-height-ratio: $line-height-base) {
```
Comment (vxsx) [inline private/sass/mixins/_functions.scss]: param names are confusing

File: private/sass/settings/_bootstrap.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
-//####################################################################################################################
-// #GLOBAL IMPORTS#
+// #############################################################################
+// GLOBAL IMPORTS
 @import "../libs/bootstrap/variables"; // using https://github.com/twbs/bootstrap-sass
```
Comment (vxsx) [inline private/sass/settings/_bootstrap.scss]: do we actually need to import them here?

File: private/sass/sites/_custom.scss [hunk 3]
```diff
@@ -30,28 +30,30 @@
...
-// #HELPERS/accessibility#
+// HELPERS/accessibility
 .skip-links {
     @extend .list-unstyled;
     position: absolute;
-    z-index: 9999;
     top: $navbar-height;
     right: 0;
     left: 0;
+    z-index: 9999;
```
Comment (vxsx) [inline private/sass/sites/_custom.scss]: can we use the z-index map?

File: tests/karma.conf.js [hunk 4]
```diff
@@ -85,7 +86,7 @@ module.exports = function (config) {
         // test results reporter to use
         // possible values: 'dots', 'progress'
         // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-        reporters: ['progress', 'coverage', 'saucelabs'],
+        reporters: ['progress', 'coverage', 'coveralls', 'saucelabs'],
```
Comment (vxsx) [inline tests/karma.conf.js]: is this reporter installed by default?

File: private/sass/mixins/_functions.scss (hunk 1 above)
Comment (FinalAngel) [inline private/sass/mixins/_functions.scss]: that's from the boilerplate

File: private/sass/settings/_bootstrap.scss (hunk 2 above)
Comment (FinalAngel) [inline private/sass/settings/_bootstrap.scss]: yep, currently we do

File: tests/karma.conf.js (hunk 4 above)
Comment (FinalAngel) [inline tests/karma.conf.js]: yes, part of the boilerplate

File: private/sass/sites/_custom.scss (hunk 3 above)
Comment (FinalAngel) [inline private/sass/sites/_custom.scss]: actually yes, this is copied from the xplorer theme, needs to be addressed there, ticket will be created

**Comment by vxsx** [comment]
:+1:

**Pull request merged**